🐞 - error TS2339: Property 'onMobileClick' does not exist on type 'TuiInputDateComponent' since 3.10.0

### Which @taiga-ui/* package(s) are the source of the bug?

addon-mobile

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/stackblitz

### Is this issue blocking you?

Blocking

### Description

error TS2339: Property 'onMobileClick' does not exist on type 'TuiInputDateComponent'.

179       this.input.onMobileClick();



since 3.10.0

### Angular version

14 or 15

### Taiga UI version

3.10.0

### Which browsers have you used?

- [ ] Chrome
- [X] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

--- a/projects/kit/components/input-date/input-date.component.ts
+++ b/projects/kit/components/input-date/input-date.component.ts
@@ -230,6 +230,15 @@ export class TuiInputDateComponent
         return this.activeItem ? '' : filler;
     }
 
+    // noinspection JSUnusedGlobalSymbols
+    /**
+     * TODO: Remove in 4.0
+     * @deprecated: use {@link onIconClick} instead
+     */
+    onMobileClick(): void {
+        this.onIconClick();
+    }
+
     onIconClick(): void {
         if (!this.computedMobile || !this.mobileCalendar) {
             return;

